### DISCUSSION TITLE
How to rotate Z only clockwise ?

### DISCUSSION THREAD

**Discussion by Jefffr** [rewbs/sd-parseq#108] [Q&A] [answered]
Hello,

I'm trying to turn the Z axis clockwise only.
How can I do this?
I want to speed up the rotation, and slow it down.

Every time I try, I get both clockwise and counter-clockwise rotation...

**Comment by rewbs** [comment, marked as answer]
Hi! Remember that Parseq uses absolute values rather than relative amounts. So you need to specify the target total rotation that you want to hit at the relevant keyframes.

Here's an example where the initial image will rotate by 45° between frame 0 and 40, another 20° between frames 41 and 80 (up to 65°), and then another 115° between frames 81 and 119 (up to 180°).

<img width="1866" alt="image" src="https://github.com/rewbs/sd-parseq/assets/74455/f01588fb-f537-4865-84f0-0dae7980c405">

Full example added to library here: https://github.com/rewbs/sd-parseq/discussions/110

Also, remember that if you rotate too much without a very high strength, the diffusion process might re-interpret the input frame as "the right way up".
